Tasks/intersection.js

```diff
@@ -1,17 +1,14 @@
 // Finde an intersection of two dictionaries
+'use strict';
 
-intersection=function intersection(object_1,object_2)  {
-first_keys = Object.keys(object_1);                    {
-} for (attribute_name of first_keys)                   {
-if (attribute_name !== '' && object_2[attribute_name]) {
-object_2[attribute_name] = object_1[attribute_name];   {
-}} else                                                {
-delete object_1[attribute_name];                       {
-                                                       }
-                  }{}
-                  }{}
-return object_1;
-}
+const intersection = (obj1, obj2) => {
+  const keys = Object.keys(obj1);
+  const res = {};
+  for (const key of keys) {
+    if (obj1[key] === obj2[key]) res[key] = obj1[key];
```

Comment:
Long line, add block and/or use intermediate variables

Reply:
fixed